### PR TITLE
docments for ImageDataLoaders.from_name_func

### PR DESCRIPTION
**Pull request by hamelsmu**
This is a super small PR, to see if I'm doing things right.  Once I get the hang of the rules, I can do more :)

cc: @muellerzr @jph00

### CODE REVIEW

File: fastai/vision/data.py [hunk 1]
```diff
@@ -112,7 +112,11 @@ def from_path_func(cls, path, fnames, label_func, valid_pct=0.2, seed=None, item
         return cls.from_dblock(dblock, fnames, path=path, **kwargs)
 
     @classmethod
-    def from_name_func(cls, path, fnames, label_func, **kwargs):
+    def from_name_func(cls,
+                       path:(str, Path), # Set the default path to a directory that a `learner` can use to save files like models.
+                       fnames, # A list of `pathlib.Path` to individual image files.
+                       label_func:callable, # A function that receives a string (the file name) and outputs a label.
+                       **kwargs) -> DataLoaders:
```
Comment (muellerzr) [inline fastai/vision/data.py:119]:
```suggestion
    def from_name_func(cls,
                       path:(str, Path), # Set the default path to a directory that a `Learner` can use to save files like models
                       fnames:list, # A list of `os.Pathlike`'s to individual image files
                       label_func:callable, # A function that receives a string (the file name) and outputs a label
                       **kwargs
                   ) -> DataLoaders:
```
Formatting nit + typing

**Review by muellerzr — CHANGES_REQUESTED** [review]
Looks pretty good for your first one! Made a few comments

**Comment by hamelsmu** [comment]
Thanks for making commit suggestions.  I ❤️ it when people do this.  Makes life so much easier

File: fastai/vision/data.py [hunk 2]
```diff
@@ -112,7 +112,12 @@ def from_path_func(cls, path, fnames, label_func, valid_pct=0.2, seed=None, item
         return cls.from_dblock(dblock, fnames, path=path, **kwargs)
 
     @classmethod
-    def from_name_func(cls, path, fnames, label_func, **kwargs):
+    def from_name_func(cls,
+                       path:(str, Path), # Set the default path to a directory that a `Learner` can use to save files like models
+                       fnames:list, # A list of `os.Pathlike`'s to individual image files
+                       label_func:callable, # A function that receives a string (the file name) and outputs a label
+                       **kwargs
+                   ) -> DataLoaders:
```
Comment (warner-benjamin) [inline fastai/vision/data.py:120]:
```suggestion
    def from_name_func(cls,
        path:(str, Path), # Set the default path to a directory that a `Learner` can use to save files like models
        fnames:list, # A list of `os.Pathlike`'s to individual image files
        label_func:callable, # A function that receives a string (the file name) and outputs a label
        **kwargs
    ) -> DataLoaders:
```

**Review by warner-benjamin — CHANGES_REQUESTED** [review]
Added a suggestion to fix formatting per the [style guide](https://github.com/muellerzr/fastai-docment-sprint/blob/master/docs_src/style-guide.md).

Otherwise looks good.

**Review by warner-benjamin — APPROVED** [review]
Looks good.

**Pull request merged**